doc/api/Buffer_Control/setMaxVideoBufferSize.md

```diff
@@ -0,0 +1,40 @@
+# setMaxVideoBufferSize
+
+## Description
+
+Set the maximum memory the video buffer can take up in the memory, in kilobytes
+Defaults at `Infinity`
+
+Once this value is reached, the player won't try to download new video
+segments anymore.
+
+This feature was designed with devices that have limited memory and trying to play very
+high bitrates tracks in minds. 
```

Comment:
Technically tracks/AdaptationSet do not have bitrate (at least in the RxPlayer API), Representation/qualities do.

Reply:
Still true